services/entity-server/src/components/solution-map/solution-map-sidebar.js

```diff
@@ -0,0 +1,115 @@
+import style from "./solution-map-sidebar.module.css";
+import React from "react";
+import { Button, Intent } from "@blueprintjs/core";
+import { Link } from "react-router-dom";
+
+const SidebarIdeaDetails = props => {
+  const { selectedIdea } = props;
+  return (
+    <div
+      className={style.ideaDetails}
+      style={{ textAlign: selectedIdea ? "left" : "center" }}
+    >
+      {!selectedIdea && (
+        <p>Select an idea in the right window to see more information! </p>
+      )}
+      {selectedIdea && (
+        <>
+          <p>
+            <b>ID:</b> <br /> {selectedIdea.idea}
+          </p>
+          <p>
+            <b>DESCRIPTION:</b> <br /> {selectedIdea.content}
+          </p>
+          <p>
+            <b>CUSTER LABEL:</b> <br /> {selectedIdea.clusterLabel}
+          </p>
+          <Link to={"/idea/" + getNameFromUri(selectedIdea.idea)}>
+            <Button minimal={true} active={true} intent={Intent.SUCCESS}>
+              More Details
+            </Button>
+          </Link>
+        </>
+      )}
+    </div>
+  );
+};
+
+const getNameFromUri = uri => {
```

Comment:
I have the feeling this will be needed several times -> move to utils namespace

Reply:
good point!